arch/power/slide_hash_power8.c

```diff
@@ -0,0 +1,71 @@
+ /* Copyright (C) 2019 Matheus Castanho <msc@linux.ibm.com>, IBM
+  * For conditions of distribution and use, see copyright notice in zlib.h
+  */
+
+#include <altivec.h>
+#include "../../zbuild.h"
+#include "../../deflate.h"
+
+static inline void slide_hash_power8_loop (deflate_state *s,
```

Comment:
I don't think this is needed.. could be wrong.

Reply:
Do you mean the includes? We need `deflate.h` for the definition of `deflate_state` and `Pos`, and removing `zbuild.h` gives some erros because `PREFIX3` is not declared when including `deflate.h`

Reply:
Simple "zbuild.h" and "deflate.h" should work as the path to source root is included in include path.

Reply:
I was referring to `static inline void slide_hash_power8_loop (deflate_state *s,`.. I don't think we need to declare it ahead since it is not being used before it is defined..

Reply:
The forward declaration was there solely for the `__attribute__((always_inline))` at the end of the line. But since the lib is already compiled with `-O2`, I guess we don't need that. Removed.